### ISSUE TITLE
How can i disable injector in debug mode, debug very trouble with injected class.

### ISSUE THREAD

**Issue by kargencW** [pamidur/aspect-injector#104]
Add MSBuild option to disable aspect injector entirely.

**Comment by pamidur** [comment]
Hi, for now there is no way to disable injections entirely. And this would be good enhancement!

For now you good disable on per injections basis or per aspect bassis:

```c#
    [Aspect(Scope.Global)]
// Disable all [Log] injections
#if !DEBUG
    [Injection(typeof(Log))]
#endif
    class Log : Attribute
    {
        [Advice(Kind.Before)]
        public void LogCall([Argument(Source.Name)] string name)
        {
            Console.WriteLine($"Calling {name}");
        }
    }
     
    class Program
    {
// Disable only this particular injections
#if !DEBUG
        [Log]
#endif
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");
        }
    }
```

**Comment by kargencW** [comment]
thank you very much

**Comment by pamidur** [comment]
I'll reopen this issue so I can track improvement of this feature: being able to disable AI completely

**Comment by pamidur** [comment]
you can now use 
```xml
  <PropertyGroup>
    <AspectInjectorEnabled>false</AspectInjectorEnabled>
  </PropertyGroup>
```
to completely disable AI for specific configurations

released in 2.2.2
